docs/source/research_references.rst

```diff
@@ -0,0 +1,14 @@
+Research References
+======================
+
+This page lists research papers that have used the Unitary Computing Compiler (UCC) in their work.
+If you have used UCC in your research, please let us know by opening an issue or pull request on GitHub to update this list.
```

Comment:
You should be able to add a link to the new issue template using syntax like this:

> https://github.com/unitaryfoundation/ucc/issues/new?template=bug_report.md